urllib3/request.py

```diff
@@ -1,13 +1,47 @@
 from __future__ import absolute_import
-try:
-    from urllib.parse import urlencode
-except ImportError:
-    from urllib import urlencode
 
 from .filepost import encode_multipart_formdata
 
+from .packages import six
 
-__all__ = ['RequestMethods']
+_Request = six.moves.urllib.request.Request
+urlencode = six.moves.urllib.parse.urlencode
```

Comment:
Can we do this as `from six.moves.urllib.request import Request` and `from six.moves.urllib.parse import urlencode`?